[DOCU-1437]Updates Immunity and StatsD exporter locations (#2821)
Kong/docs.konghq.com: app/enterprise/2.1.x/vitals/vitals-prometheus-strategy.md

### Download and configure StatsD exporter

> The latest release of StatsD exporter can be pulled from Docker Hub with the
> following command: 

Reviewer: ```suggestion
StatsD exporter is distributed as a Docker image. Pull the latest version
with the following command: 
```

```sh
docker pull kong/statsd-exporter-advanced:0.3.1
```

> The binary is distributed with some specific features like min/max gauges
> and Unix domain socket support.

Reviewer: ```suggestion
The binary includes features like min/max gauges and Unix domain 
socket support.
```
I know you didn't write or edit this line, but adjusting it because I added the suggestion to use "distributed" above. Won't get into trying to figure out what "some specific features" really means, that'll go too deep down a rabbit hole.
